src/core/producer/stage.cpp

```diff
@@ -106,8 +109,8 @@ struct stage::impl : public std::enable_shared_from_this<impl>
                 auto& tween = tweens_[std::get<0>(transform)];
                 auto  src   = tween.fetch();
                 auto  dst   = std::get<1>(transform)(tween.dest());
-                tweens_[std::get<0>(transform)] =
-                    tweened_transform(src, dst, std::get<2>(transform), std::get<3>(transform));
+                auto  index = std::get<0>(transform);
+                tweens_[index] = tweened_transform(src, dst, std::get<2>(transform), std::get<3>(transform));
```

Comment:
DOes this actually change anything or is it just cosmetics?

Reply:
Cosmetic. I meant to remove that bit as it is from the other way I tried. 
Do you want me to change it back?

Reply:
Na, it's fine, just wanted to make sure